client/galaxy/style/less/reports.less

```diff
@@ -207,3 +213,8 @@
 .quarter_width {
     width: 25%;
 }
+
+.toolFormBody {
```

Comment:
Sorry about that. I'll fix it tomorrow. I guess I'll just do some extra div wrapping.